src/Forside/Sammenligningspanel/SektorPanel/SektorPanel.tsx

```diff
@@ -0,0 +1,17 @@
+import React, { FunctionComponent } from 'react';
+import Sykefraværsprosentpanel, { SykefraværprosentpanelProps } from '../Sykefraværsprosentpanel/Sykefraværsprosentpanel';
+import SammenligningsIkon from '../SammenligningsIkon';
+import { SykefraværshistorikkType } from '../../../api/sykefraværshistorikk';
+
+const SektorPanel: FunctionComponent<SykefraværprosentpanelProps> = props => {
+    return (
+        <Sykefraværsprosentpanel
+            laster={props.laster}
+            sykefraværprosentLabel={props.sykefraværprosentLabel}
+            sykefraværsprosent={props.sykefraværsprosent}
+            ikon={<SammenligningsIkon label={SykefraværshistorikkType.SEKTOR} />}
```

Comment:
Forslag: Hva med å legge svg-filen direkte inn her? Og droppe SammenligningsIkon-komponenten. Den blir ikke brukt andre steder enn i disse panel-klassene uansett :)

Reply:
delvis enig, på den måte blir det litt lettere å håndtere når vi bytter svg fil for eks. særlig hvis vi kanskje bruker den flere steder. Men det er mulig også.